libs/velo-external-db-core/src/web/wix_data_facade.ts

```diff
@@ -0,0 +1,33 @@
+import { errors } from '@wix-velo/velo-external-db-commons'
+const { UnauthorizedError } = errors
+import axios from 'axios'
+import { decodeBase64 } from '../utils/base64_utils'
+
+type PublicKeyResponse = {
+    publicKey: string;
+};
+
+export interface WixDataFacade {
```

Comment:
Our convention is that Interfaces to classes should have the same name as the class only with i at the beginning
So the interface for the `WixDataFacade` class should be named `IWixDataFacade`